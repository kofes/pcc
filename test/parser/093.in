const
	r = 10;
var
	l : real = 2*pi*r;
begin
end
