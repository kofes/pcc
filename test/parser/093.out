<function table>
</function table>
<type table>
   <type>BOOLEAN</type>
   <type>CHAR</type>
   <type>INTEGER</type>
   <type>REAL</type>
</type table>
<var table>
   <var>L:V:REAL:62.831850</var>
   <var>PI:C:REAL:3.1415926535897932385</var>
   <var>R:C:INTEGER:10</var>
</var table>
<block statement>
</block statement>
